Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityInputSystem.cs

```diff
@@ -119,6 +120,16 @@ public interface IMixedRealityInputSystem : IMixedRealityEventSystem
         /// </summary>
         void ClearFallbackInputStack();
 
+        /// <summary>
+        /// Interface for subscribing handlers to a specific phase of the event propagation
+        /// </summary>
+        void RegisterPropagationHandler<T>(IEventSystemHandler handler, PropagationPhase phase = PropagationPhase.BubbleUp) where T : IEventSystemHandler;
```

Comment:
It might be good to rewrite this the same way `IMixedRealityEventPropagationData` was broken out into its own interface. Otherwise this is a breaking change.

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/BreakingChanges.html#the-breaking-change-adds-value-but-could-be-written-in-a-way-that-isnt-breaking

Reply:
Breaking up this and adding a InputPropagationSystem property to CoreSystem as main API for registration.